ContentProtectionSampleApp/app/src/main/java/com/ooyala/sample/players/CustomActivity.java

```diff
@@ -0,0 +1,88 @@
+package com.ooyala.sample.players;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.ArrayAdapter;
+import android.widget.Button;
+import android.widget.CheckBox;
+import android.widget.EditText;
+import android.widget.Spinner;
+import  android.widget.Toast;
+import com.ooyala.sample.R;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+
+public class CustomActivity extends Activity {
+    public static String getName() {
+        return "Custom Activity";
+    }
+    private EditText embedCodeEditText;
+    private EditText pCodeEditText;
+    private EditText apiKeyEditText;
+    private EditText secretEditText;
+    private EditText accountIdEditText;
+    private String embedCode;
+    private String pCode;
+    private String apiKey;
+    private String secret;
+    private String accountId;
+    private CheckBox autoPlayCheckBox;
+    private boolean autoPlay;
+
+  @Override
+    public void onCreate(Bundle savedInstanceState) {
+      super.onCreate(savedInstanceState);
+      setContentView(R.layout.embed_pcode_layout);
+      embedCodeEditText = (EditText) findViewById(R.id.embed_edit_text);
```

Comment:
It is now redundant to cast the type of the object when using `findViewById` you can remove the casting type now